index.js

```diff
@@ -1497,6 +1498,25 @@
     return xBool !== yBool ? or(x, y) : xEmpty;
   });
 
+  //# ifElse :: (a -> Boolean) -> (a -> b) -> (a -> b) -> a -> b
+  //.
+  //. Takes a unary predicate, a unary "if" function, a unary "else"
+  //. function, and a value of any type, and returns the result of
+  //. applying the "if" function to the value if the value satisfies
+  //. the predicate; the result of applying the "else" function to the
+  //. value otherwise.
+  //.
+  // ```javascript
```

Comment:
I just spent some time investigating a README rendering issue. Spot the missing character. ;)